algo/src/main/kotlin/com/hartwig/actin/algo/evaluation/molecular/MolecularRuleMapper.kt

```diff
@@ -306,8 +311,8 @@ class MolecularRuleMapper(resources: RuleMappingResources) : RuleMapper(resource
 
     private fun proteinHasLimitedExpressionByIHCCreator(): FunctionCreator {
         return { function: EligibilityFunction ->
-            val (expressionLevel, protein) = functionInputResolver().createOneStringOneIntegerInput(function)
-            ProteinHasLimitedExpressionByIHC(protein, expressionLevel)
+            val (expressionLevel, protein) = functionInputResolver().createOneProteinOneIntegerInput(function)
+            ProteinHasLimitedExpressionByIHC(expressionLevel, protein)
```

Comment:
These are backwards (protein is the first field). They could be flipped around...or is it better to use `with`?
```suggestion
            with (functionInputResolver().createOneProteinOneIntegerInput(function)) {
                ProteinHasLimitedExpressionByIHC(proteinName, integer)
            }
```

Reply:
Thank-you, not sure how that happened.